styles.css

```diff
@@ -0,0 +1,29 @@
+header {
+    position: fixed;
+    top: 0px;
+    width: 100%;
+    height: 19px;
+    padding: 25px 0px;
+}
+@media screen and (max-width: 600px) {
+    div.nav-bar {
+        display: none;
+    }
+} 
+
+
+@media only screen and (min-width: 601px) {
+    nav.navbar {
+      font-size: 80px;
+    }
+  }
+  body {
+    display:flex
+  }
+
+
+    @media (max-width: 600px) {

```

Comment:
Good work so far! 👍 
Please be sure to use the correct indentation and spacing in your styles.cc file.